globals.go

```diff
@@ -14,6 +14,7 @@ const (
 	HostRunning         = "running"
 	HostTerminated      = "terminated"
 	HostUninitialized   = "initializing"
+	HostSpawning        = "spawning"
```

Comment:
Let's not call it this since it may cause confusion for spawnhosts. Since only containers will be in this state for a noticeable period of time, how about "downloading"?

Reply:
@john-m-liu I changed it to "building" because containers stay in this state because they always have to build container images with the agent on it using docker build, and that only sometimes involves downloading images. Let me know if you think this is still unclear.